### ISSUE TITLE
required_when with false

### ISSUE THREAD

**Issue by mohamed-hendawy** [poppinss/indicative#145]
when use required_when with field it's value false it does not work 

```
{
  field2: 'required_when:field1,false'
}
```

field 2 is not required if field1 value is false

**Comment by thetutlage** [comment]
Will add it

**Comment by mohamed-hendawy** [comment]
@thetutlage  thanks and i wait it :D

**Comment by thetutlage** [comment]
Ohh the `required_when` rule already exists. Docs here http://indicative.adonisjs.com/#indicative-schema-rules

Just search for `required_when`

**Comment by mohamed-hendawy** [comment]
i know that required_when rule already exists but there a bug in it  try to use it with true or false like this

```js
{
  field2: 'required_when:field1,false'
}
```

when field1 is false field2 does not requierd

**Comment by thetutlage** [comment]
Fixed in 4.0
